deployed v1 to joint account, tested cf stack and agent creation

languo-aws opened the pull request:
*Issue #, if available:*

*Description of changes:*


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

languo-aws commented:
@brianloyal cleaned up output and minor updates

hasanp87 commented:
@languo-aws Dockerfile includes specific accountID , I assume you want to annonymize and ask user to fill it themselves or auto-populate from an environment variable ? 
Also, the notebook /README must contain example questions to ask the agent

hasanp87 commented:
@languo-aws "To proceed, I need the following information: 1. The seed protein sequence you want to optimize 2. A name for your workflow run 3. The S3 output location where results should be stored 4. The workflow ID to use 5. The container image URI for protein design 6. The IAM role ARN for execution permissions Could you please provide these details so I can help you optimize your protein sequence?" - can we automate some of the config so that we can limit the user inputs to include 1 , 2 and 3 - the user questions should not be technical, rather specific to the task at hand

languo-aws commented:
@hasanp87 I modified the cf stack (which spares the user from having to provide things related to the cf resources such as "workflow ID, container image URI, IAM role ARN".  Added a new lambda function to monitor workflows after triggered.
Also added usage examples in a README.

hasanp87 commented:
just noticed a discrepancy though, the readme says "The template protein_design_stack.yaml creates all necessary resources including:
• Amazon Bedrock Agent with action groups •" . Seems like the template does not do this and you need to run the notebook for it , correct ? can you update the README accordingly
